#General parameters
n_sector: 8
delta_time: 0.10
linear_threshold: 0.01
max_distance_obstacle: 1.3
agent_radius: 0.25